src/angular/tooltip/tooltip-directives.ts

```diff
@@ -0,0 +1,44 @@
+import { Directive, ElementRef, inject, Input, numberAttribute } from '@angular/core';
+
+@Directive({
+  selector: '[sbb-tooltip]',
+})
+export class SbbTooltipDirective {
+  #elementRef = inject<ElementRef<HTMLElement>>(ElementRef);
+
+  @Input('sbb-tooltip')
+  public set tooltip(value: string | null) {
+    if (value) {
+      this.#elementRef.nativeElement.setAttribute('sbb-tooltip', value);
+    } else {
+      this.#elementRef.nativeElement.removeAttribute('sbb-tooltip');
+    }
+  }
+  public get tooltip(): string | null {
+    return this.#elementRef.nativeElement.getAttribute('sbb-tooltip');
+  }
+
+  @Input({ alias: 'sbb-tooltip-open-delay', transform: numberAttribute })
+  public set openDelay(value: number) {
+    if (!isNaN(value) && value >= 0) {
```

Comment:
Slight misunderstanding; `null` should still be accepted to reset the value.
```suggestion
  public set openDelay(value: number | null) {
    if (value != null && !isNaN(value) && value >= 0) {
```